endpoint/build-legato.sh

```diff
@@ -12,5 +12,8 @@ then
 fi
 source $COMMON_FILE
 
-download_legato_repo
-build_legato_repo
+if [ ! -d "legato" ]
+then 
+    download_legato_repo
```

Comment:
What if `repo` did run but it failed unexpectedly? Thus, there would be one `legato` directory, but in vain.

Reply:
If it failed unexpectedly. The user should remove the legato directory and download again.

Reply:
You have to recover and manipulate the expected repo sync, otherwise it causes confusing.